setup.py

```diff
@@ -39,7 +39,7 @@
     long_description=open('README.rst').read(),
     author='Kenneth Reitz',
     author_email='me@kennethreitz.com',
-    url='http://certifi.io/',
+    url='https://github.com/certifi/python-certifi',
```

Comment:
Now it's back up, use the original URL with HTTPS:
```suggestion
    url='https://certifi.io/',
```